roles/rabbitmq/tasks/main.yml

```diff
@@ -10,7 +10,7 @@
     dest: '{{ rabbitmq_config_file }}'
     insertafter: 'Network Connectivity'
     regexp: '^\s*{tcp_listeners,'
-    line: "    {tcp_listeners, [{% for interface in rabbitmq_interface%}{\"{{interface}}\", {{rabbitmq_port}}}{% if loop.index != loop.length %},{% endif %}{% endfor %}]}"
+    line: "    {tcp_listeners, [{% set comma = joiner(\",\") %}{% for interface in rabbitmq_interface %}{% if interface %}{{ comma() }}{\"{{interface}}\", {{ rabbitmq_port }}}{% endif %}{% endfor %}]}"
```

Comment:
This is complicated enough that I almost wonder if we should just ship a custom filter plugin to handle convering `rabbimq_interface` into the appropriate format.